high-availability.helmfile.yaml

```diff
@@ -31,7 +31,7 @@ releases:
         value: false
   - name: ratify
     namespace: gatekeeper-system
-    chart: charts/ratify/
+    chart: charts/ratify/ # CHANGE to 'ratify/ratify' before copying to high-availability.helmfile.yaml
```

Comment:
the comment is a bit confusing to me, when do we need to make this change?

Reply:
This change is made as a pre release item. I can add that to comment